pylint_odoo/test/main.py

```diff
@@ -62,6 +62,7 @@
     'attribute-string-redundant': 33,
     'renamed-field-parameter': 2,
     'deprecated-data-xml-node': 5,
+    'resource-not-exist': 4,
```

Comment:
the old test.yml is a non-exists file too

https://github.com/oca/pylint-odoo/blob/master/pylint_odoo/test_repo/broken_module/test.yml

Jesus
Could you delete this reference since that you add a correct name ?